### ISSUE TITLE
Benchmark is wildly inaccurate despite PR provided

### ISSUE THREAD

**Issue by jhsware** [marko-js/isomorphic-ui-benchmarks#15]
### Details

Isomorphic benchmark is wildly inacurate, despite respectul contribution of ~working~ PR from other framework maintainer.
https://github.com/marko-js/isomorphic-ui-benchmarks/pull/10

### Expected Behavior

Merging contributed PR and updating README (Marko is still insanely fast). If you publish a benchmark and someone provides a PR which updates the implementation of one of the frameworks. The least you are obliged to do is to merge the pull request...

### Actual Behavior

Duck and cover... absolute silence... FOR A YEAR, come on that's not just dragging your feet... ;)

### Possible Fix

Just merge... it won't take a minute…
https://github.com/marko-js/isomorphic-ui-benchmarks/pull/10
oh... and update the readme :)

Good luck!

**Comment by DylanPiercey** [comment]
Updated to include the latest versions of all libraries and an optimized version of Inferno.

Unfortunately that PR was _not_ complete, and _not_ in a state that could be merged since as the author said it wasn't working. There were many things in that PR that were unrelated to the inferno changes as well.

Also there were many things that needed to be updated in this repo, `rollup`, `babel`, `webpack` and each individual library as well. Since the benchmark was last run/updated on March 3, 2017 and things move quickly this was quite a bit more work than just merging and updating the README. These upgrades ended up being necessary to fairly benchmark each package since many frameworks ship with babel plugins, webpack loaders, etc which are required to be updated in order to use the latest version.

` come on that's not just dragging your feet... ;)` I feel is an unnecessary insinuation that we were intentionally not updating this benchmark to "Make us look good" or something of that nature. That is not true, as I've described above this was more effort than it may have appeared and also had to be prioritized. I also believe that anyone viewing this benchmark was able to clearly see the date it was last run and conclude that the numbers may no longer be totally accurate.

We definitely took longer to update this than I'd like to see. Thanks for your attempt to keep us honest.
